src/wopiserver.py

```diff
@@ -146,15 +146,15 @@ def init(cls):
                 except OSError:
                     cls.log.error('msg="Failed to open the provided certificate or key to start in https mode"')
                     raise
-            cls.wopiurl = cls.config.get('general', 'wopiurl')
-            cls.homepath = cls.config.get('general', 'homepath', fallback='/home/username')
-            cls.recoverypath = cls.config.get('io', 'recoverypath', fallback='/var/spool/wopirecovery')
+            cls.wopiurl = cls.config.get('general', 'wopiurl').strip('/')
+            cls.homepath = cls.config.get('general', 'homepath', fallback='/home/username').strip('/')
+            cls.recoverypath = cls.config.get('io', 'recoverypath', fallback='/var/spool/wopirecovery').strip('/')
```

Comment:
@glpatcern are you sure that `strip` does the right thing?

According to https://www.w3schools.com/python/ref_string_strip.asp, it removes trailing and leading slashes in our case. Which makes the path relative instead of absolute. I noticed it because I have a readonly filesystem in our Kubernetes setup and it tries to write somewhere it should not.

Reply:
You're right, sorry for that! Let's not over-engineer this, I will simply revert the change for the OS-level paths (where extra slashes are irrelevant anyway) and leave it for URLs (where the protocol prefix MUST be present)